scapy/fields.py

```diff
@@ -283,6 +283,10 @@ class ThreeBytesField(X3BytesField, ByteField):
     def i2repr(self, pkt, x):
         return ByteField.i2repr(self, pkt, x)
 
+class SignedCharField(Field):
```

Comment:
Maybe you could rename this field as `SignedByteField` (since the other fields are named using `Byte`)